homeassistant/components/speedtestdotnet/const.py

```diff
@@ -13,24 +14,35 @@
 
 SPEED_TEST_SERVICE: Final = "speedtest"
 
-SENSOR_TYPES: Final[tuple[SensorEntityDescription, ...]] = (
-    SensorEntityDescription(
+
+@dataclass
+class SpeedtestSensorEntityDescription(SensorEntityDescription):
+    """Class describing Speedtest sensor entities."""
+
+    value: Callable = round
+
+
+SENSOR_TYPES: Final[tuple[SpeedtestSensorEntityDescription, ...]] = (
+    SpeedtestSensorEntityDescription(
         key="ping",
         name="Ping",
         native_unit_of_measurement=TIME_MILLISECONDS,
         state_class=STATE_CLASS_MEASUREMENT,
+        value=lambda value: round(value, 2),
```

Comment:
Does the ping value need two decimal places? IMO, rounding without decimals is sufficient.
SpeedTest site also rounding ping value without decimals

![image](https://user-images.githubusercontent.com/478555/134697870-474205bf-2fd1-48ad-85a6-906541c39c96.png)

Reply:
I agree, lets add this change:

```suggestion
        value=lambda value: round(value),
```

Reply:
`value` is [`round`](https://github.com/home-assistant/core/blob/ece1e4f502dd298341fe93d4f4f16ce81be85e12/homeassistant/components/speedtestdotnet/const.py#L22) by default so I think we don't need to override `value`.